### DISCUSSION TITLE
Can I mix font styles in the middle of a text?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [ocornut/imgui#5107] [New Users Build/Link/Run issues ONLY!]
I would like to do something like:
![image](https://user-images.githubusercontent.com/20190653/158058305-bbcc331b-2ba1-456a-840b-9c81debe4c74.png)
To outline keyboard shortcuts in my OSD interactive manual

**Comment by ocornut** [comment]
You can't but you can emit different Text() or draw_list->AddText() calls with cursor positioning and font changes.

**Reply by ItsCubeTime** [reply]
Thanks for the quick reply

> you can emit different Text() or draw_list->AddText() calls with cursor positioning and font changes.

Can you elaborate how I would/could utilize this for my use case?

**Reply by julienta** [reply]
You can search on how to use addText function in the imgui_draw.cpp. Like Ocornut said you can pass it a font, size and position. But then you are in charge of updating/maintaining the text placement. Since it's low level.
You will need to build and combine multiple fonts. Here is a link on how to do that:
https://github.com/ocornut/imgui/blob/master/docs/FONTS.md
